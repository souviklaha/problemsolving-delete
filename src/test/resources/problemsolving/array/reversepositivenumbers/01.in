11
4 3 8 9 -2 6 10 13 -1 2 3