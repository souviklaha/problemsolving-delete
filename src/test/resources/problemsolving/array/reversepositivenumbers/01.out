9 8 3 4 -2 13 10 6 -1 3 2